zhaquirks/legrand/wire_pilot.py

```diff
@@ -0,0 +1,130 @@
+"""Module for Legrand Cable Outlet with pilot wire functionality."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import add_to_registry_v2
+import zigpy.types as t
+from zigpy.zcl import ClusterType
+from zigpy.zcl.foundation import (
+    BaseAttributeDefs,
+    BaseCommandDefs,
+    Direction,
+    ZCLAttributeDef,
+    ZCLCommandDef,
+)
+
+from zhaquirks.legrand import LEGRAND, MANUFACTURER_SPECIFIC_CLUSTER_ID
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID_2 = 0xFC40  # 64576
+
+
+WIRE_PILOT_MODE_ATTR = 0x4000
+
+LEGRAND_HEAT_MODE_ATTR = 0x00
+
+
+class LegrandCluster(CustomCluster):
+    """LegrandCluster."""
+
+    cluster_id = MANUFACTURER_SPECIFIC_CLUSTER_ID
+    name = "LegrandCluster"
+    ep_attribute = "legrand_cluster"
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        device_mode = ZCLAttributeDef(
+            id=0x0000,
+            type=t.data16,
+            is_manufacturer_specific=True,
+        )
+        led_dark = ZCLAttributeDef(
+            id=0x0001,
+            type=t.Bool,
+            is_manufacturer_specific=True,
+        )
+        led_on = ZCLAttributeDef(
+            id=0x0002,
+            type=t.Bool,
+            is_manufacturer_specific=True,
+        )
+        wire_pilot_mode = ZCLAttributeDef(id=WIRE_PILOT_MODE_ATTR, type=t.Bool)
+
+    async def write_attributes(self, attributes, manufacturer=None) -> list:
+        """Write attributes to the cluster."""
+
+        attrs = {}
+        for attr, value in attributes.items():
+            attr_def = self.find_attribute(attr)
+            attr_id = attr_def.id
+            if attr_id == WIRE_PILOT_MODE_ATTR:
+                attrs[0x0000] = [0x02, 0x00] if value else [0x01, 0x00]
```

Comment:
0x0000 -> `LegrandCluster.AttributeDefs.device_mode.id`

Reply:
What do the byte arrays represent? We should name them at a minimum.

Reply:
I don't really known why it uses array instead of boolean. My assumption is that legrand is using same attribute for various devices.